src/plugins/widgets/trello/Trello.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Props, defaultData, TrelloList, TrelloData } from "./types";
+import { pull_trello_data } from "./api";
+import "./Trello.sass";
+
+const Trello: React.FC<Props> = ({cache, data = defaultData, setCache}) => {
+    React.useEffect(() => {
+        pull_trello_data(data.settingsData).then(setCache);
```

Comment:
We prefer for the naming convention to be uniform throughout the project. So `camelCase` naming standard and `getAPIAction` is preferred here. 
```suggestion
        getTrelloData(data.settingsData).then(setCache);
```